board/app/controllers/thread_controller.php

```diff
@@ -1,107 +1,149 @@
 <?php
+
 class ThreadController extends AppController {
 
-    const MAX_THREAD_PER_PAGE = 7;
-    const MAX_COMMENT_PER_PAGE = 5;
-  
-    /*
-    * Create new thread
-    * :: - STATIC FUNCTION, can be called from the class name
-    * -> - INSTANCE, can only be called from an instance of the class.
-    */
-    public function create()
+const CREATE_THREAD = 'create';
+const CREATE_THREAD_END = 'create_end';
+
+/*
+* Create new thread
+* :: - STATIC FUNCTION, can be called from the class name
+* -> - INSTANCE, can only be called from an instance of the class.
+*
+* Everything inputted on the form (view/thread/create.php) will be gathered by this function
+*/
+
+   public function create() 
     {
         $thread = new Thread();
         $comment = new Comment();
-        $current_page = Param::get('page_next', 'create');
-      
-        switch ($current_page) {
-            case 'create':
-            break;
-      
-        case 'create_end':
-            $thread->title = Param::get('title');
-            $comment->username = Param::get('username');
-            $comment->body = Param::get('body');
- 
-            try 
-            {
-                $thread->create($comment);
-            } catch (ValidationException $e) {
-                $current_page = 'create';
-            }
-              break;
+        $current_page = Param::get('page_next', self::CREATE_THREAD);   
+                    
+        switch ($current_page) { 
+            case self::CREATE_THREAD:
+                break;
+            /*  
```

Comment:
It might be better to either remove the comment, or summarize it to point out important points only po :bow: